pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.uid2</groupId>
     <artifactId>uid2-shared</artifactId>
-    <version>4.3.0-dddd3edffd</version>
+    <version>4.3.2-SNAPSHOT</version>
```

Comment:
I thought this gets committed as part of the pipeline

Reply:
This was the pipeline 👍